CHANGELOG.md

```diff
@@ -17,6 +15,8 @@ For information on changes in released versions of Teku, see the [releases page]
 ## Unreleased Changes
 
 ### Breaking Changes
+- The command argument `--Xengine-exchange-capabilities` has been removed, update to use `--engine-exchange-capabilities` if you are using this option.
```

Comment:
I believe the new command is `--exchange-capabilities-enabled`.

Reply:
so the deprecated i copied was wrong? i can check...